Packs/HPEArubaClearpass/Integrations/HPEArubaClearpass/HPEArubaClearpass.py

```diff
@@ -0,0 +1,472 @@
+
+    human_readable = f"HPE Aruba Clearpass attribute with ID: {attribute_id} deleted successfully."
+    return CommandResults(readable_output=human_readable)
+
+
+def get_active_sessions_list_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    Gets a list of active sessions. If session_id was not given, all the active sessoions will be returned.
+    """
+    session_id = args.get('session_id')
+    device_ip = args.get('device_ip')
+    device_mac_address = args.get('device_mac_address')
+    state = args.get('state')
+    visitor_phone = args.get('visitor_phone')
+
+    session_filter = {}
+    session_filter.update({"id": session_id}) if session_id else None
+    session_filter.update({"framedipaddress": device_ip}) if device_ip else None
+    session_filter.update({"mac_address": device_mac_address}) if device_mac_address else None
+    session_filter.update({"state": state}) if state else None
+    session_filter.update({"visitor_phone": visitor_phone}) if visitor_phone else None
+    session_filter = json.dumps(session_filter)  # the API requires the value of 'filter' to be a json object.
+    params = {"filter": json.dumps(session_filter)}
+
+    res = client.prepare_request(method='GET', params=params, url_suffix='session')
+    readable_output, all_active_sessions_list = parse_items_response(res, active_sessions_response_to_dict)
+    outputs = [active_sessions_response_to_dict(item) for item in all_active_sessions_list]
+    human_readable = tableToMarkdown('HPE Aruba Clearpass Active Sessions', readable_output, removeNull=True)
+
+    return CommandResults(
+        readable_output=human_readable,
+        outputs_prefix='HPEArubaClearpass.sessions.list',
+        outputs_key_field='id',
+        outputs=outputs,
+    )
+
+
+def active_sessions_response_to_dict(response: dict) -> dict:
+    return {
+        'ID': response.get('id'),
+        'Device_IP': response.get('framedipaddress'),
+        'Device_mac_address': response.get('mac_address'),
+        'State': response.get('state'),
+        'Visitor_phone': response.get('visitor_phone', False)
+    }
+
+
+def disconnect_active_session_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    Disconnects an active session by the session_id which is a mandatory field.
+    """
+    session_id = args.get('session_id')
+    url_suffix = f"/session/{session_id}/disconnect"
+    body = {"id": session_id, "confirm_disconnect": True}
+
+    res = client.prepare_request(method='POST', params={}, url_suffix=url_suffix, body=body)
+    outputs = {"Error_code": res.get('error'), "Response_message": res.get('message')}
+    human_readable = tableToMarkdown('HPE Aruba Clearpass Disconnect active session', outputs, removeNull=True)
+
+    return CommandResults(
+        readable_output=human_readable,
+        outputs_prefix='HPEArubaClearpass.sessions.disconnect',
```

Comment:
Why under disconnect? Shouldn't we just update the sessions path that the session has disconnected?